forge-gui/res/cardsfolder/upcoming/emperor_of_bones.txt

```diff
@@ -0,0 +1,12 @@
+Name:Emperor of Bones
+ManaCost:1 B
+Types:Creature Skeleton Noble
+PT:2/2
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ At the beginning of combat on your turn, exile up to one target card from a graveyard.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TargetMin$ 0 | TargetMax$ 1 | ValidTgts$ Card | TgtPrompt$ Select up to one target card from a graveyard 
+K:Adapt:2:1 B
+T:Mode$ CounterAddedOnce | ValidCard$ Card.Self | TriggerZones$ Battlefield | CounterType$ P1P1 | Execute$ TrigChangeZone | TriggerDescription$ Whenever one or more +1/+1 counters are put on CARDNAME, put a creature card exiled with CARDNAME onto the battlefield under your control with a finality counter on it. It gains haste. Sacrifice it at the beginning of the next end step.
+SVar:TrigChangeZone:DB$ ChangeZone | ChangeType$ Creature.ExiledWithSource | Origin$ Exile | Hidden$ True | Destination$ Battlefield | WithCountersType$ FINALITY  | AnimateSubAbility$ Animate
```

Comment:
wrong animate use

Reply:
"under your control" also means you need `GainControl` Param